### ISSUE TITLE
Aligned media insert not working properly

### ISSUE THREAD

**Issue by boonebgorges** [cuny-academic-commons/social-paper#30]
From http://commons.gc.cuny.edu/groups/cac-community-team-project-planning/docs/social-paper-beta-2-directory-page-and-writing-papers/#comment-2030:

> The add media did not work for me when I chose to align left or right with a large image. Just got a blank box with a caption. When I set align center it worked.

There seem to be a non-trivial number of issues with media. Should we just disable it for v1?

**Comment by mkgold** [comment]
> Should we just disable it for v1?

If needed. I would like to see more testing on this, however, before we take that step, as it would be a significant feature loss.

**Comment by r-a-y** [comment]
We could also remove or hide the align buttons if they are causing issues for the media toolbar.

**Comment by sheesh** [comment]
I agree with Matt that it would be a significant feature loss; I found the
easy ability to align the image very impressive and useful and did not
experience any issues.

I tried to replicate this bug by uploading an extremely large image. The
result was an error message. I think most users will know not to upload
large images.

erin rose glass | digital fellow | cuny grad center | nyc
www.erinroseglass.com | @erinroseglass

**Comment by christianwach** [comment]
I couldn't replicate this locally. Nor could I replicate on cdev when I created a new paper there. Anyone have an example of the problem?
